test/io/CsvReaderTest.ooc

```diff
@@ -68,6 +68,24 @@ CsvReaderTest: class extends Fixture {
 			filename free()
 			reader free()
 		})
+		this add("skip header", func {
+			filename := t"test/io/input/semicolondelimiter.csv"
+			reader := CsvReader open(filename, ';', true)
+			rowCounter := 0
+			for (row in reader) {
+				for (i in 0 .. row count) {
+					rowString := row[i] toString()
+					correctAnswer := ((i + 4) + rowCounter * 3) toString()
```

Comment:
This test is identical to the one above except `4` instead of `1` here, since we skipped one row.